features/support/finance_helpers.rb

```diff
@@ -76,7 +77,7 @@ def check_payment_confirmation_message(amount)
 
 def adjust_charge(amount, random_number)
   # Start from payments page and add a positive or negative charge
```

Comment:
Suggest amending the comment, or removing it entirely:

```suggestion
  # Start from any logged in screen with the appropriate user, and add a positive or negative charge
```